iocore/net/UnixNetAccept.cc

```diff
@@ -369,7 +384,15 @@ NetAccept::do_blocking_accept(EThread *t)
     // Assign NetHandler->mutex to NetVC
     vc->mutex = h->mutex;
     localt->schedule_imm(vc);
-  } while (loop);
+  } while (count <= additional_accepts);
+
+  // for dedicated blocking accept threads, if we reach the maximum additional accepts
```

Comment:
Is this useful? In the current case this can be set to the equivalent of "0" and clearly doesn't require this logic. Why is is needed now?